components/microsoft_outlook/microsoft_outlook.app.mjs

```diff
@@ -1,11 +1,72 @@
+import axios from "axios";
```

Comment:
You should be using the sdk library instead of `axios` directly. Look at this example of how can you [use the sdk](https://github.com/PipedreamHQ/pipedream/pull/2084/files#diff-340fd3b3d79c473ce390d17b136cf79241039a5b423b98f1bde04a814b39411fR131-R172).

Look the reason [here](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries).

Reply:
Using `import { axios } from "@pipedream/platform";` instead